app/controllers/petitions_controller.rb

```diff
@@ -1,5 +1,5 @@
 class PetitionsController < ApplicationController
```

Comment:
Starting a thread here re: /network_controller removal.

Sprint 2 is `Latest petitions that are created from your network` and in the Node.js exercise, there is also a `/network` stub [here](https://github.com/quaelin/node_api_exercise/blob/main/routes/network.js#L8). I believe the intention is that the intention here is that candidates could use `/network` to build sprint 2. I think you're off today, so for now I'll create a PR off your branch to bring back `/network` and tomorrow we could discuss either keeping it in both or removing in both.